src/ape/_cli.py

```diff
@@ -28,13 +29,19 @@ def display_config(ctx, param, value):
 class ApeCLI(click.MultiCommand):
     _commands = None
 
+    def invoke(self, ctx: click.Context) -> Any:
+        try:
+            return super().invoke(ctx)
+        except ApeException as err:
+            raise Abort(str(err))
```

Comment:
@fubuloubu

Reply:
To preserve the stack trace
```suggestion
            raise Abort(str(err)) from err
```

Reply:
TIL